src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -204,7 +204,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             _optionCompareText = optionCompareText
             _embedVbCoreRuntime = embedVbCoreRuntime
             _suppressEmbeddedDeclarations = suppressEmbeddedDeclarations
-            _parseOptions = parseOptions
+            _parseOptions = If(parseOptions, VisualBasicParseOptions.Default) ' WHY: 
```

Comment:
There are occasions when the `parseOptions` is null, which can cause problems. So assign a default implementation.